tests/js/setup-globals.js

```diff
@@ -15,6 +15,7 @@ global.wcSettings = {
 	date: {
 		dow: 0,
 	},
+	has_filterable_products: true,
```

Comment:
@thealexandrelara Thanks for writing the tests 🙌🏻 I am still getting familiar with the way we write tests, but I think it should be camelCase instead of snake_case to be consistent with other field names in this file.

@tjcafferkey What do you think? 🙂

Reply:
@imanish003 I believe @thealexandrelara is setting this setting value (https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/blocks/attribute-filter/block.tsx#L86-L90) which is what the block uses internally to decide whether to render or not. This value gets set server-side (https://github.com/woocommerce/woocommerce-blocks/blob/trunk/src/BlockTypes/AllProducts.php#L24-L25) and passed to the client to tell the filter there is in fact products there which are filterable.

This was done because filter blocks can be used in Classic themes (non-block themes) such as Storefront in widget areas which means they would get rendered on pages where no products would be present (such as a T&C's page for example). So we set this value so we can return `null` in the filter when there are no products available to render.

Reply:
@tjcafferkey Ah! Got it. Thanks for explaining that 🙏🏻